src/main/java/com/beust/jcommander/JCommander.java

```diff
@@ -197,27 +197,32 @@ public JCommander() {
      * @param object The arg object expected to contain {@link Parameter} annotations.
      */
     public JCommander(Object object) {
-        this();
-        addObject(object);
+        this(object, (ResourceBundle) null);
     }
 
     /**
      * @param object The arg object expected to contain {@link Parameter} annotations.
      * @param bundle The bundle to use for the descriptions. Can be null.
      */
     public JCommander(Object object, @Nullable ResourceBundle bundle) {
-        this(object);
-        setDescriptionsBundle(bundle);
+        this(object, bundle, (String[]) null);
     }
 
     /**
      * @param object The arg object expected to contain {@link Parameter} annotations.
      * @param bundle The bundle to use for the descriptions. Can be null.
      * @param args The arguments to parse (optional).
      */
-    public JCommander(Object object, ResourceBundle bundle, String... args) {
-        this(object, bundle);
-        parse(args);
+    public JCommander(Object object, @Nullable  ResourceBundle bundle, String... args) {
+        this();
```

Comment:
It is as otherwise `this.options` remains uninitialised.

You might confuse it with the optionally calling `super()` in constructors of subclasses.